src/GafferUI/BackdropNodeGadget.cpp

```diff
@@ -147,15 +147,14 @@ std::string BackdropNodeGadget::getToolTip( const IECore::LineSegment3f &line )
 
 	if( title.size() )
 	{
-		result += "<h3>" + title + "</h3>";
+		result += "# " + title;
```

Comment:
For this and Boxes with custom descriptions, if the description begins with an H1, the tooltip will contain two H1s in sequence, which is potentially confusing from a readability standpoint. The proper semantic solution would be to mark up node titles as `<title>` (boo hiss HTML, but in this case it's a composed string...) and style that class to look different from H1s.

Reply:
Following that argument, the correct semantics for the plug headings is also `<title>`, right?

Reply:
> the correct semantics for the plug headings is also `<title>`, right?

Ugh, I shouldn't have typed `<title>`, that was incorrect. Wrong headspace. Semantic top-level titles in a "multi-section page" (which Gaffer would be considered) are an ongoing issue in HTML, without an agreed solution.

To actually answer your question: yes, whatever we choose, it would be best if they had the same markup. Let's deal with this in the follow-up styling PR, where I'll have time to explore Qt's stylesheet.